src/lib/README

```diff
@@ -1,2 +1,4 @@
+# libraries
```

Comment:
Rename file to .md if you want to use markdown

Reply:
done